test/e2e/user_management_test.go

```diff
@@ -138,14 +143,7 @@ func (s *userManagementTestSuite) TestUserDeactivation() {
 		// We cannot wait days for testing deactivation so for the purposes of the e2e tests we use a hack to change the provisioned time
 		// to a time far enough in the past to trigger auto deactivation. Subtracting the given period from the current time and setting this as the provisioned
 		// time should test the behaviour of the deactivation controller reconciliation.
-		manyManyDaysAgo := 999999999999999
-		durationDelta := time.Duration(manyManyDaysAgo) * time.Hour * 24
-		updatedProvisionedTime := &metav1.Time{Time: time.Now().Add(-durationDelta)}
-		murMember1, err = s.hostAwait.UpdateMasterUserRecordStatus(murMember1.Name, func(mur *toolchainv1alpha1.MasterUserRecord) {
-			mur.Status.ProvisionedTime = updatedProvisionedTime
-		})
-		require.NoError(s.T(), err)
-		s.T().Logf("masteruserrecord '%s' provisioned time adjusted", murMember1.Name)
+		murMember1, updatedProvisionedTime := s.setOldCreationTimestamp(murMember1.Name)
```

Comment:
it would probably be more explicit or understandable when reading the code here if the `manyManyDaysAgo` value was an argument of the `setOldCreationTimestamp` method.
But it's not a blocker.

Reply:
It will always be called with the same value - I don't see any benefit for doing it - that's why I just copied it and called the function `setOldCreationTimestamp`

Reply:
ok, fair enough. As I said, it's just a cosmetic change anyways